transpiler/src/test/resources/org/polystat/py2eo/transpiler/simple-tests/arithmetic-conversion/conversion-3.yaml

```diff
@@ -0,0 +1,8 @@
+disabled: True
+python: |
+  def test():
+    num_int = 123
+    num_flo = 1.23
+  
+    num_new = num_int * num_flo
+    return num_new.__class__ is float
```

Comment:
The implementation of the is operator will most likely appear later than arithmetic (maybe not appear at all), so it’s better to replace it with `==` here, and send these to tests for 6.10.3